src/content/en/2020/caching.md

```diff
@@ -8,19 +8,19 @@ authors: [roryhewitt, raghuramakrishnan71]
 reviewers: [csswizardry, jzyang, jaisanth, Soham-S-Sarkar]
 analysts: [raghuramakrishnan71]
 translators: []
-roryhewitt_bio: Enterprise Architect at <a href="https://www.akamai.com/">Akamai</a>. Passionate about performance. Competition-level snowboarder. Multiple black belts. Motorcyclist. Boxer. Troublemaker. 
+roryhewitt_bio: Enterprise Architect at <a href="https://www.akamai.com/">Akamai</a>, who is passionate about performance. A British ex-patriate, he has lived in San Francisco for more than twenty years. In his spare time, he's a long-distance adventure motorcyclist, competition-level snowboarder and boxer with multiple black belts in various styles of karate. He likes being known as a troublemaker. Most importantly, he's a father and husband and the owner of Luna the cat. Find him at <a href="https://www.linkedin.com/in/roryhewitt/">LinkedIn</a>.
 raghuramakrishnan71_bio: Enterprise architect at <a href="https://www.tcs.com/">Tata Consultancy Services</a>, working on large digital transformation programs in the public sector. A technology enthusiast with a special interest in performance engineering. An avid traveler, intrigued by astronomy, history, biology, and advancements in medicine. A strong follower of the 47th verse, Chapter 2 of Bhagavad Gita "karmaṇy-evādhikāras te mā phaleṣhu kadāchana" meaning "You have a right to perform your prescribed duty, but you are not entitled to the fruits of action."
 
 discuss: 2056
 results: https://docs.google.com/spreadsheets/d/1fYmpSN3diOiFrscS75NsjfsrKXzxxhUMNcYSqXnQJQU/
 queries: 20_Caching
-#featured_quote: Caching benefits both the end users (they get their web pages quickly) and the companies serving the web pages (reducing the load on their servers). Caching really is a win-win!
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
+#featured_quote: Caching benefits both end users (they get their web pages quickly) and the companies serving web pages (reducing the load on their servers). Caching really is a win-win!
+#featured_stat_1: 25.6
+#featured_stat_label_1: Number of HTTP responses don't include any caching information, which means they're probably being cached wrongly
+#featured_stat_2: 21.4%
+#featured_stat_label_2: Number of responses that aren't using revalidation, resulting in wasted bandwidth and increased latency
+#featured_stat_3: 78.6%
+#featured_stat_label_3: Number of sites could save up to 2MB of bandwidth by improving their caching policy
```

Comment:
Can activate these now:

```suggestion
featured_quote: Caching benefits both end users (they get their web pages quickly) and the companies serving web pages (reducing the load on their servers). Caching really is a win-win!
featured_stat_1: 25.6%
featured_stat_label_1: Number of HTTP responses don't include any caching information, which means they're probably being cached wrongly
featured_stat_2: 21.4%
featured_stat_label_2: Number of responses that aren't using revalidation, resulting in wasted bandwidth and increased latency
featured_stat_3: 78.6%
featured_stat_label_3: Number of sites could save up to 2MB of bandwidth by improving their caching policy
```

Also this is what it looks like now:

![Featured quote](https://user-images.githubusercontent.com/10931297/101750638-3494c500-3ac7-11eb-96e1-b099ffe871dc.png)

The quotes itself is quite short compared to the labels.

Reply:
I'd also suggest rewording "Number" to "Percent"